pkg/movies/routes.go

```diff
@@ -0,0 +1,24 @@
+package movies
+
+import (
+	"github.com/labstack/echo"
+	"github.com/nathankhuu/goyagi/pkg/application"
+)
+
+// RegisterRoutes takes in an Echo router and registers routes onto it.
+func RegisterRoutes(e *echo.Echo, app application.App) {
+	g := e.Group("/movies")
+
+	// RegiserRoutes takes in an application.App struct and we wrap it around an
+	// internal handler struct. This allows us to create a methods over the
+	// handler struct that act as our endpoint handler functions. Doing this
+	// allows us to access our App struct (which contains references to all
+	// objects that need to only be instantiated once) from all of our handlers.
+	// If we ever need to add new dependencies to our App struct, we'll
+	// automatically make them available to all our handlers through this
+	// construct.
+	h := handler{app}
```

Comment:
Was curious as to why we don't need to import the `handler` pkg then I realized this is all included in the `movies` pkg.